notification-manager/src/main/resources/application.yaml

```diff
@@ -1,3 +1,16 @@
 spring:
   application:
-    name: notification-manager
\ No newline at end of file
+    name: notification-manager
+  mail:
+    host: smtp.gmail.com
+    port: 587
+    username: sukharevroman96@gmail.com
+    properties:
+      mail:
+        debug: true
+        transport:
+          protocol: smtp
+        smtp:
+          starttls:
+            enable: true
+          auth: true
```

Comment:
I'm not sure whether you need `auth` field. What does it stand for?

Reply:
it tells spring to use smtp protocol, when i send message, but i don't start application without it. This property was written in the guide, so i wrote it